src/impala/emit.cpp

```diff
@@ -31,15 +31,15 @@ class CodeGen : public IRBuilder {
         return Def();
     }
 
```

Comment:
I don't like this change.
Makes calling this function more complicated.
Also: you don't gain much by letting the callee create the `true`/`false` literal. Since literals are cse-d anyway, the location will be in 99% bogus anyway.